app/models/discourse_gamification/gamification_score.rb

```diff
@@ -37,6 +37,26 @@ def self.calculate_scores(since_date: Date.today, only_subclass: nil)
         SET score = EXCLUDED.score;
       SQL
     end
+
+    def self.merge_scores(source_user, target_user)
+      DB.exec(<<~SQL, source_id: source_user.id, target_id: target_user.id)
+        WITH new_scores AS (
+          SELECT 8722 AS user_id, date, SUM(score) AS score
+          FROM gamification_scores
+          WHERE user_id IN (27384, 8722)
```

Comment:
I don't think you intended to hardcode ids here right?

Reply:
:facepalm: